File: php/app/8-setterGetter.php
<?php

class Person {
    private $name;
    private $age;

    function __construct($name, $age) {
        $this->name = $name;
        $this->age = $age;
    }

    public function setName ($name) {
        $this->name = $name;
    }

    public function getName () {
        echo "Get Name: " . $this->name . "\n";
    }
}

    $Person1 = new Person("Hera", 20);
    $Person1->getName();
    $Person1->setName("Hero");
    $Person1->getName();

?>

File: golang/app/8-setterGetter.go
package main

import "fmt"

type Person struct {
	Name string
	Age int
}

func main()  {
	var person1 = Person{"Hera", 20}
	person1.getName();
	person1.setName("Hero")
	person1.getName();
}

func (p *Person) getName()  {
	fmt.Println("Get name: ", p.Name)
}

func (p* Person) setName(name string) {
	p.Name = name
}

